docker-image/flink/Dockerfile

```diff
@@ -0,0 +1,102 @@
+# Ubuntu 16.04 (Xenial)
```

Comment:
can we update this model off my new aztk spark vanilla dockerfile - in the branch: feature/new_containers

Reply:
I took a look at that file, It has some Spark specific environment variables being set. I'm not sure what we should do because I feel like we should have a base Aztk image that all software build off of. So we would have `Ubuntu -> Aztk -> $Software Vanilla -> $Software Specific`, but I think to do that we have to go through the hoop of building base images.